chromium_src/chrome/installer/setup/install_worker.cc

```diff
@@ -32,17 +32,18 @@
 #if BUILDFLAG(ENABLE_BRAVE_VPN)
 #include "brave/components/brave_vpn/browser/connection/ikev2/win/brave_vpn_helper/brave_vpn_helper_constants.h"
 #include "brave/components/brave_vpn/browser/connection/ikev2/win/brave_vpn_helper/brave_vpn_helper_state.h"
+#include "brave/components/brave_vpn/browser/connection/wireguard/win/brave_vpn_wireguard_service/service_constants.h"
 
 namespace {
 
-base::FilePath GetBraveVPNServicePath(const base::FilePath& target_path,
-                                      const base::Version& version) {
+base::FilePath GetBraveVPNHelperPath(const base::FilePath& target_path,
+                                     const base::Version& version) {
   return target_path.AppendASCII(version.GetString())
       .Append(brave_vpn::kBraveVPNHelperExecutable);
 }
 
-bool ConfigureBraveVPNServiceAutoRestart(const base::FilePath& exe_path,
-                                         const CallbackWorkItem&) {
+bool InstallBraveVPNService(const base::FilePath& exe_path,
```

Comment:
nit: `kBraveVpnHelperInstall` constants name could be renamed to `kBraveVpnServiceInstall`